README.md

```diff
@@ -1,2 +1,6 @@
-# SentryOne
```

Comment:
This page should be generic it's just a copy and paste of the other readme. Over time this repo should have all kinds of stuff in it relating to SentryOne, so maybe remove the conditions part?